src/renderer/entities/network/lib/chainsService.ts

```diff
@@ -9,103 +9,101 @@ import chainsDev from '@renderer/assets/chains/chains_dev.json';
 import { ChainId } from '@renderer/domain/shared-kernel';
 import { getRelaychainAsset, nonNullable, totalAmount, ZERO_BALANCE } from '@renderer/shared/lib/utils';
 import { Balance } from '@renderer/entities/asset/model/balance';
-import { ChainLike, IChainService } from './common/types';
+import { ChainLike } from './common/types';
 import { isKusama, isPolkadot, isTestnet, isNameWithNumber } from './common/utils';
 
 const CHAINS: Record<string, any> = {
   chains: chainsProd,
   'chains-dev': chainsDev,
 };
 
-export function useChains(): IChainService {
-  const getChainsData = (): Chain[] => {
-    return CHAINS[process.env.CHAINS_FILE || 'chains'];
-  };
-
-  const getChainById = (chainId: ChainId): Chain | undefined => {
-    const chainsData: Chain[] = CHAINS[process.env.CHAINS_FILE || 'chains'];
-
-    return chainsData.find((chain) => chain.chainId === chainId);
-  };
-
-  const getStakingChainsData = (): Chain[] => {
-    const chainsData: Chain[] = CHAINS[process.env.CHAINS_FILE || 'chains'];
-
-    return chainsData.reduce<Chain[]>((acc, chain) => {
-      if (getRelaychainAsset(chain.assets)) {
-        acc.push(chain);
-      }
-
-      return acc;
-    }, []);
-  };
-
-  const sortChains = <T extends ChainLike>(chains: T[]): T[] => {
-    let polkadot;
-    let kusama;
-    const testnets = [] as T[];
-    const parachains = [] as T[];
-    const numberchains = [] as T[];
-
-    chains.forEach((chain) => {
-      if (isPolkadot(chain.name)) polkadot = chain;
-      else if (isKusama(chain.name)) kusama = chain;
-      else if (isTestnet(chain.options)) testnets.push(chain);
-      else if (isNameWithNumber(chain.name)) numberchains.push(chain);
-      else parachains.push(chain);
-    });
+export const chainsService = {
```

Comment:
This syntax allows to use `ChainsService` in more performant way
With ability to be mocked with ease
With clearer context in code
```typescript
// code ...
chainsService.getChainsData();
// code ...
```